Server/mods/deathmatch/logic/CConsoleCommands.cpp

```diff
@@ -1175,29 +1175,15 @@ bool CConsoleCommands::WhoIs(CConsole* pConsole, const char* szArguments, CClien
         // Any player requested?
         if (strcmp(szArguments, "*") == 0)
         {
-            // Iterate the players and echo their IPs and ports if anyone was requested
-            unsigned int                   uiCount = 0;
-            CPlayerManager*                pPlayerManager = pConsole->GetPlayerManager();
-            CPlayer*                       pPlayer;
-            list<CPlayer*>::const_iterator iter = pPlayerManager->IterBegin();
-            for (; iter != pPlayerManager->IterEnd(); iter++)
+            if (pConsole->GetPlayerManager()->HasAnyJoined())
```

Comment:
Does it still work if the player hasn't joined completely, but is already connected?

Reply:
No. I should add a method to check if there are any players connected.